### PR TITLE
Adjust editor error codes names and descriptions.

### PR DESCRIPTION
**Pull request by Mati365**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Internal (core): Adjust editor error codes and associated descriptions.

---

### Additional information

Closes https://github.com/cksource/ckeditor5-commercial/issues/6761#issue-2634762995

### CODE REVIEW

File: packages/ckeditor5-core/src/editor/editor.ts [hunk 1]
```diff
@@ -384,22 +384,22 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
...
-				 * [https://ckeditor.com/contact/](https://ckeditor.com/contact/) or order a trial at
-				 * [https://orders.ckeditor.com/trial/premium-features](https://orders.ckeditor.com/trial/premium-features).
+				 * The `licenseKey` property is missing in the editor configuration.
 				 *
-				 * If you do not use premium features, add the `'GPL'` license key instead.
+				 * 	* If you are using the editor in a commercial setup, please provide your license key.
+				 * 	* If you still need to acquire a key, please contact us or start a free trial.
+				 * 	* If you are using the editor under a GPL license or another license from our Open Source Initiative ,
+				 * 	  use the 'GPL' license key instead.
 				 *
```
Comment (Witoso) [inline packages/ckeditor5-core/src/editor/editor.ts:393]: The section above misses the links from the GDoc.

**Review by Witoso — CHANGES_REQUESTED** [review]

File: packages/ckeditor5-core/src/editor/editor.ts (hunk 1 above)
Comment (Mati365) [reply packages/ckeditor5-core/src/editor/editor.ts:393]: Whoops, missed that, good point

Comment (Mati365) [reply packages/ckeditor5-core/src/editor/editor.ts:394]: Fixed https://github.com/ckeditor/ckeditor5/pull/17400/commits/1c3bb05e44e76009a0bf7106feaa02bf17a91360

File: packages/ckeditor5-core/src/editor/editor.ts [hunk 2]
```diff
@@ -384,22 +384,23 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
...
-				 * The `licenseKey` is missing in the editor configuration. If you use premium features,
-				 * please provide your license key. If you do not have a key yet, please contact us at
-				 * [https://ckeditor.com/contact/](https://ckeditor.com/contact/) or order a trial at
-				 * [https://orders.ckeditor.com/trial/premium-features](https://orders.ckeditor.com/trial/premium-features).
+				 * The `licenseKey` property is missing in the editor configuration.
 				 *
-				 * If you do not use premium features, add the `'GPL'` license key instead.
+				 * 	* If you are using the editor in a commercial setup, please provide your license key.
+				 * 	* If you still need to acquire a key, please [contact us](https://ckeditor.com/contact/) or
+				 *	  [start a free trial](https://portal.ckeditor.com/checkout?plan=free).
```
Comment (Witoso) [inline packages/ckeditor5-core/src/editor/editor.ts:391]:
```suggestion
				 *	  [create a free account with a 14 day premium features trial](https://portal.ckeditor.com/checkout?plan=free).
```

Just noticed we are not mentioning free accounts, just free trials.

Comment (Mati365) [reply packages/ckeditor5-core/src/editor/editor.ts]: Fixed https://github.com/ckeditor/ckeditor5/pull/17400/commits/6ac6f1e53cbc8ff64874d2be942c2f2ac6fa8571

File: packages/ckeditor5-core/src/editor/editor.ts [hunk 3]
```diff
@@ -384,22 +384,23 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
...
-				 * please provide your license key. If you do not have a key yet, please contact us at
-				 * [https://ckeditor.com/contact/](https://ckeditor.com/contact/) or order a trial at
-				 * [https://portal.ckeditor.com/checkout](https://portal.ckeditor.com/checkout?plan=free).
+				 * The `licenseKey` property is missing in the editor configuration.
 				 *
-				 * If you do not use premium features, add the `'GPL'` license key instead.
+				 * 	* If you are using the editor in a commercial setup, please provide your license key.
+				 * 	* If you still need to acquire a key, please [contact us](https://ckeditor.com/contact/) or
+				 *	  [create a free account with a 14 day premium features trial](https://portal.ckeditor.com/checkout?plan=free).
+				 * 	* If you are using the editor under a GPL license or another license from our Open Source Initiative ,
```
Comment (Witoso) [inline packages/ckeditor5-core/src/editor/editor.ts:392]:
```suggestion
				 * 	* If you are using the editor under a GPL license or another license from our Open Source Initiative,
```

**Review by Witoso — APPROVED** [review]

**Pull request merged**